balancer-js/src/modules/pools/pool-types/concerns/composableStable/join.concern.ts

```diff
@@ -1,6 +1,30 @@
-import { JoinPoolParameters, JoinConcern, JoinPoolAttributes } from '../types';
+import {
```

Comment:
Sorry, I think I confused things by suggesting we use this. I didn't realise it wasn't already implemented. Lets remove this and add in a separate PR later.